apps/web/src/views/PositionManagers/hooks/useTotalAssetInSingleDepositTokenAmount.ts

```diff
@@ -0,0 +1,18 @@
+import { useMemo } from 'react'
+import { formatAmount } from '@pancakeswap/utils/formatFractions'
+import { Currency, CurrencyAmount } from '@pancakeswap/sdk'
+
+export const useTotalAssetInSingleDepositTokenAmount = (
+  singleDepositTokenAmount?: CurrencyAmount<Currency>,
+  ohterTokenAmount?: CurrencyAmount<Currency>,
+  singleDepositTokenPriceUSD?: number,
+  ohterTokenPriceUSD?: number,
+) => {
+  const totalAssetInSingleDepositTokenAmount = useMemo(() => {
+    return (
+      Number(formatAmount(singleDepositTokenAmount) ?? 0) +
+      (Number(formatAmount(ohterTokenAmount) ?? 0) * (ohterTokenPriceUSD ?? 0)) / (singleDepositTokenPriceUSD ?? 0)
+    )
```

Comment:
For the number calculation can u use `BigNumber` to do it?

Reply:
got it, will fix it later